client/layout/masterbar/logged-out.jsx

```diff
@@ -75,6 +76,7 @@ class MasterbarLoggedOut extends PureComponent {
 
 	renderSignupItem() {
 		const { currentQuery, currentRoute, sectionName, translate } = this.props;
+		const signupFlow = get( currentQuery, 'signup_flow' );
```

Comment:
Super minor but we could move this declaration closer to the code that actually uses it.